onnxruntime/core/graph/model.cc

```diff
@@ -82,10 +82,18 @@ Model::Model(const std::string& graph_name,
     opset_id_proto->set_version(domain.second);
   }
 
+  std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*> model_local_funcs;
+  for (auto& func : model_local_functions) {
+    auto func_ptr = model_proto_.add_functions();
+    func_ptr->CopyFrom(func);
```

Comment:
One minor concern I have with the API is the need for copying FunctionProtos. I feel an alternative API that encourages a coding style that doesn't require such copies would be better. But, given the deadline, I guess it is better to stick to the current solution.